### DISCUSSION TITLE
How to make page title center using Set-PnPPage on SharePoint Online?

### DISCUSSION THREAD

**Discussion by in8AnkKu** [pnp/powershell#960] [Q&A] [answered]
Hi,

I need to update the page title text along with the alignment (center). I had tried to do it with `Set-PnpPage` command, but using this, i am able to update the title not alignment. Please suggest me how can i make the page title center? Page is modern page in communication site.
Appreciate any help the community can provide.

![image](https://user-images.githubusercontent.com/50485498/127828511-79e61b01-f9eb-4585-a7e5-9840456a6297.png)

**Comment by LeonArmston** [comment, marked as answer]
Hi

The below will allow you to check and then change the page header on a modern page to be centre aligned.

```powershell
#Get Page Object
$page = Get-PnPPage -Identity "Centre.aspx"
$page.PageHeader #Check Text Alignment Property for existing value

#Update Page Header Text Alignment to Center
$page.PageHeader.TextAlignment =  "Center"
$page.Save()
Invoke-PnPQuery

```

Thanks

**Reply by in8AnkKu** [reply]
Thanks a lot :), Its working as expected.
